### PR TITLE
refactored DataSource configuration

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

File: src/main/resources/db.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# MySql properties
+db.driver = com.mysql.cj.jdbc.Driver
+db.url = jdbc:mysql://localhost:3306/spring_hw?serverTimezone=UTC
+db.username = root
+db.password = Vi4ft5q7Ktrr9T6toPcN
+# Hibernate properties
```
Comment (matviichuk92) [inline src/main/resources/db.properties:6]: I think we should delete these comments

**Review by matviichuk92 — APPROVED** [review]
good job!

File: src/main/java/spring/intro/config/AppConfig.java [hunk 2]
```diff
@@ -3,26 +3,33 @@
...
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "spring.intro.service",
         "spring.intro.dao"
 })
 
 public class AppConfig {
+    @Autowired
+    private Environment env;
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/config/AppConfig.java]:
```suggestion
    private Environment environment;
```

File: src/main/resources/db.properties (hunk 1 above)
Comment (RomanDubovskyi) [reply src/main/resources/db.properties:6]: > I think we should delete these comments

it looks ok here

**Review by RomanDubovskyi — COMMENTED** [review]
improve PR name a bit, change "add solution" to refactored DataSource configuration or alternative

File: src/main/java/spring/intro/config/AppConfig.java (hunk 2 above)
Comment (Victoria-Moiseienko) [reply src/main/java/spring/intro/config/AppConfig.java]: Done

File: src/main/java/spring/intro/config/AppConfig.java [hunk 3]
```diff
@@ -3,26 +3,33 @@
...
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "spring.intro.service",
         "spring.intro.dao"
 })
 
 public class AppConfig {
+    @Autowired
+    private Environment environment;
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/config/AppConfig.java:22]: better to use constructor autowiring

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**